cmab/training_data/reset_Q.py

```diff
@@ -0,0 +1,2 @@
+Q_path = "training_data\\Q.json"
+num_games_path = "training_data\\num_games.json"
```

Comment:
newline at end

Reply:
Yes, sorry for that this folder should have been deleted before the push